src/tags/case.ts

```diff
@@ -31,7 +31,11 @@ export default class extends Tag {
       })
       .on('tag:else', () => (p = this.elseTemplates))
```

Comment:
I think we can set p to [] which serves as a blackhole when `elaeTemplates` is already set. And we need initialize it as null instead of empty string. For case

```
{%else%}{%else%}we don't want this output
```

Reply:
I used the `p = []` and an `else` counter to prevent rendering any branches that follow the `else`.

> `{%else%}{%else%}we don't want this output`

I added an empty else block to the [tests](https://github.com/joel-hamilton/liquidjs/blob/e2b22d0187916a8d31ad7772248a4668f85424d0/test/integration/tags/unless.spec.ts#L60) and it passed without changing the initialization. Let me know if you see another edge case I'm missing though.